ui/components/app/confirm-gas-display/confirm-legacy-gas-display/__snapshots__/confirm-legacy-gas-display.test.js.snap

```diff
@@ -42,6 +44,7 @@ exports[`ConfirmLegacyGasDisplay should match snapshot 1`] = `
       >
         <h6
           class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography typography--h6 typography--weight-normal typography--style-normal typography--color-text-alternative"
+          data-testid=""
```

Comment:
@dan437 Why are these empty `data-testid=""` here?

Reply:
I've added an empty string as a default value based on this: https://github.com/MetaMask/metamask-extension/pull/19169#discussion_r1229766775

However, it seems it would be better to keep it as I had it initially.